src/analysis/inner_product_space/basic.lean

```diff
@@ -1437,8 +1472,11 @@ lemma is_bounded_bilinear_map_inner [normed_space ℝ E] :
     by simp only [← algebra_map_smul 𝕜 r y, algebra_map_eq_of_real, inner_smul_real_right],
   bound := ⟨1, zero_lt_one, λ x y,
     by { rw [one_mul], exact norm_inner_le_norm x y, }⟩ }
+
+
```

Comment:
Intentional whitespace?

Reply:
I kept one of them intentionally, the end of a section was directly after the end of a proof.